test/util/testnode/rpc_client.go

```diff
@@ -66,10 +66,12 @@ func StartGRPCServer(logger log.Logger, app srvtypes.Application, appCfg *srvcon
 		return nil, Context{}, emptycleanup, err
 	}
 
-	err = srvgrpc.StartGRPCServer(cctx.goContext, logger, appCfg.GRPC, grpcSrv)
-	if err != nil {
-		return nil, Context{}, emptycleanup, err
-	}
+	go func() {
+		// StartGRPCServer is a blocking function, we need to run it in a go routine.
+		if err := srvgrpc.StartGRPCServer(cctx.goContext, logger, appCfg.GRPC, grpcSrv); err != nil {
+			panic(err)
```

Comment:
bit of a hack, we can refactor to handle this correctly, but I don't know if it's super important since this would just fail during test setup either way.